.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
nit: Committing the eclipse files into version control is generally not a recommended idea. I think `.classpath` is okay, but `.project` is something that can be generated directly in Eclipse I believe.

Consider at least adding `.project` to `.gitignore` if it can be generated directly in eclipse.